routes/api/index.js

```diff
@@ -0,0 +1,9 @@
+const router = require("express").Router();
+const usersAndTasksRoutes = require("./usersAndTasks");
```

Comment:
You should keep the name of the file as users.js. In the future we might need to add other routes to this file and the name of "usersAndTasks" might not make sense.